### PR TITLE
Starts a change log

### PR DESCRIPTION
**Pull request by MDaubs**
Letting folks know what changed is only polite.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Version 0.3.0 (in development)
```
Comment (aprescott) [inline CHANGELOG.md]: I know we've already planned for a 0.3.0 release, but for the purposes of this heading, should we avoid referencing version numbers on the off-chance that they change? For example, if we released v1.6 and decided that the next version is tentatively 1.6.1 (in development), but later decide it needs to be 1.7, we'd be misleading people about the existence of a 1.6.1 release. Maybe we can just not specify it here?

**Comment by aprescott** [comment]
Other than my fairly minor comment, this looks great to me!

File: CHANGELOG.md (hunk 1 above)
Comment (MDaubs) [inline CHANGELOG.md]: Agreed, thanks!

**Comment by aprescott** [comment]
:heart:

**Pull request merged**